src/python_framework/nhd_network_traversal.py

```diff
@@ -2,34 +2,60 @@
 A demonstration version of this code is stored in this Colaboratory notebook:
     https://colab.research.google.com/drive/1ocgg1JiOGBUl3jfSUPCEVnW5WNaqLKCD
 
+
+## command line input order [debuglevel = 0,verbose = true,showtiming = true, supernetworks = supernetwork_options]
 """
 import nhd_network_utilities as nnu
 import recursive_print
 import time
 import os
+import sys
 
-# NOTE: these methods can lose the "connections" and "rows" arguments when
-# implemented as class methods where those arrays are members of the class.
-
+try:
+    arg1 = int(sys.argv[1])
+    arg2 = sys.argv[2]
+    arg3 = sys.argv[3]
+    arg4 = sys.argv[4]
+except:
+    arg1 = 0
+    arg2 = True
+    arg3 = True
+    arg4 = 'Brazos_LowerColorado_ge5'
+    supernetwork_options = {
+            'Pocono_TEST1'
+            ,'Pocono_TEST2'
+            , 'LowerColorado_Conchos_FULL_RES'
+            , 'Brazos_LowerColorado_ge5'
+            , 'Brazos_LowerColorado_FULL_RES'
+            , 'Brazos_LowerColorado_Named_Streams'
+            , 'CONUS_ge5'
+            , 'Mainstems_CONUS'
+            , 'CONUS_Named_Streams'
+            , 'CONUS_FULL_RES_v20'
+        }
+    printout1 = 'command line arguements available: debuglevel = {-3,-2,-1, 0}, verbose = {True/False}, showtiming = {True/False}, supernetwork = ' + str(supernetwork_options)
+    
 def main():
     # find the path of the test scripts, several levels above the script path
     root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
     test_folder = os.path.join(root, r'test')
     
     supernetworks = {}
-    supernetworks.update({'Pocono_TEST1':{}})
-    supernetworks.update({'LowerColorado_Conchos_FULL_RES':{}}) 
-    supernetworks.update({'Brazos_LowerColorado_ge5':{}}) ##NHD Subset (Brazos/Lower Colorado)"""
-    supernetworks.update({'Brazos_LowerColorado_FULL_RES':{}}) 
-    supernetworks.update({'Brazos_LowerColorado_Named_Streams':{}}) 
-    supernetworks.update({'CONUS_ge5':{}}) ##NHD CONUS order 5 and greater"""
-    supernetworks.update({'Mainstems_CONUS':{}})
-    supernetworks.update({'CONUS_Named_Streams':{}})
-    supernetworks.update({'CONUS_FULL_RES_v20':{}}) # = False
+    supernetworks.update({arg4:{}})
+    # supernetworks.update({'Pocono_TEST1':{}})
+    # supernetworks.update({'LowerColorado_Conchos_FULL_RES':{}}) 
+    # supernetworks.update({'Brazos_LowerColorado_ge5':{}}) ##NHD Subset (Brazos/Lower Colorado)"""
+    # supernetworks.update({'Brazos_LowerColorado_FULL_RES':{}}) 
+    # supernetworks.update({'Brazos_LowerColorado_Named_Streams':{}}) 
+    # supernetworks.update({'CONUS_ge5':{}}) ##NHD CONUS order 5 and greater"""
+    # supernetworks.update({'Mainstems_CONUS':{}})
+    # supernetworks.update({'CONUS_Named_Streams':{}})
+    # supernetworks.update({'CONUS_FULL_RES_v20':{}}) # = False
```

Comment:
This has the ability to process more than one `supernetwork` at a time. Can we preserve that in the command-line input somehow?